tools/npz2scn.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+
+# npz2scn - tool for extracting semantic scene information from 3dscenegraph
+#           dataset (https://3dscenegraph.stanford.edu/)
+
+import argparse
+import json
+import sys
+
+import numpy as np
+
+
+# Convert ndarrays to python lists so that we can serialize.
+# Tranform coordinates by rotating Y-axis to Z-axis
+def fixCoords(entry):
```

Comment:
Based on naming convention:
```suggestion
def fix_coords(entry):
```